pkg/framework/clusterautoscalers.go

```diff
@@ -0,0 +1,21 @@
+package framework
+
+import (
+	"context"
+	"fmt"
+
+	caov1 "github.com/openshift/cluster-autoscaler-operator/pkg/apis/autoscaling/v1"
```

Comment:
nit: I would have expected this to be imported as `autoscalingv1` (normally it's the groups shortname followed by a version), though that may clash with the upstream autoscaling APIs 🤔

Reply:
thanks, that makes sense to me. iirc i copied this import from another piece of code. but for future reference i will try to make these import more idiomatic.